cypress/integration/404.spec.ts

```diff
@@ -23,5 +23,3 @@ it('should display custom 404 page when not found', () => {
   cy.findByRole('link', { name: 'Or go to homepage' }).click()
   cy.url().should('equal', `${Cypress.config().baseUrl}/`)
 })
-
-export {}
```

Comment:
This weird `export {}` is not needed anymore. Check the `cypress/tsconfig.json` changes for more info.